object-cache.php

```diff
@@ -295,6 +330,12 @@ function delete($id, $group = 'default') {
 	}
 
 	function flush() {
+
+		// Return true is flushing is disabled
+		if ( ! WP_MEMCACHE_FLUSHING ) {
```

Comment:
(I know I've had issues with flushing during testing before, just curious as to the specifics of this one.)